src/app/payment/paymentMethod/StripePaymentMethod.tsx

```diff
@@ -3,65 +3,96 @@ import React, { useCallback, FunctionComponent } from 'react';
 import { Omit } from 'utility-types';
 
 import { withCheckout, CheckoutContextProps } from '../../checkout';
-import { TranslatedString } from '../../locale';
+import {  TranslatedString } from '../../locale';
 
 import HostedWidgetPaymentMethod, { HostedWidgetPaymentMethodProps } from './HostedWidgetPaymentMethod';
+import StripeV3CustomCardForm from './StripeV3CustomCardForm';
 
 export type StripePaymentMethodProps = Omit<HostedWidgetPaymentMethodProps, 'containerId'>;
 
 export interface StripeOptions {
     alipay?: StripeElementOptions;
     card: StripeElementOptions;
+    cardCvc: StripeElementOptions;
+    cardExpiry: StripeElementOptions;
+    cardNumber: StripeElementOptions;
     iban: StripeElementOptions;
     idealBank: StripeElementOptions;
 }
 interface WithCheckoutStripePaymentMethodProps {
     storeUrl: string;
 }
-
-export enum StripeV3PaymentMethodType {
+export enum StripeElementType {
     alipay = 'alipay',
     card = 'card',
+    cardCvc = 'cardCvc',
+    cardExpiry = 'cardExpiry',
+    cardNumber = 'cardNumber',
     iban = 'iban',
     idealBank = 'idealBank',
 }
-
 const StripePaymentMethod: FunctionComponent<StripePaymentMethodProps & WithCheckoutStripePaymentMethodProps> = ({
       initializePayment,
       method,
       storeUrl,
       ...rest
   }) => {
-    const paymentMethodType = method.id as StripeV3PaymentMethodType;
-    const additionalStripeV3Classes = paymentMethodType !== StripeV3PaymentMethodType.alipay ? 'optimizedCheckout-form-input widget--stripev3' : '';
+    const cardFormIndividualElements = method.initializationData.cardFormIndividualElements;
```

Comment:
This flag got renamed to `shouldUseIndividualCardFields`  Please read [this](https://github.com/bigcommerce/checkout-sdk-js/pull/985#discussion_r494689199) comment